test/helpers/cmd.go

```diff
@@ -85,7 +85,7 @@ func (b *CmdStreamBuffer) Filter(filter string) (*FilterBuffer, error) {
 
 	err := json.Unmarshal(b.Bytes(), &data)
 	if err != nil {
-		return nil, fmt.Errorf("could not parse JSON from command %q", b.Cmd())
+		return nil, fmt.Errorf("could not parse JSON from command %q\n%s\n%s", b.Cmd(), err, b.Bytes())
```

Comment:
Sorry one more nit, `%w` for errors

Reply:
That doesn't seem consistent across the codebase. We're using `%s`, `%v`, and `%w` according to `git grep -P "fmt.Errorf.*, err"`. https://github.com/golang/go/wiki/CodeReviewComments also points to `%v`.

I'm happy to change but not sure to understand the logic. I also don't think it will scale if we rely solely on reviews to catch this.

Reply:
@pchaigno It's [relatively new](https://blog.golang.org/go1.13-errors#TOC_3.3.) and it seems like the code review doc hasn't been updated for it :/

Unfortunately, yes there's inconsistent in the codebase as it sometimes slips by. In this case, consistency is not better--we should just use the correct format specifier when we can :)

Doubly unfortunate in terms of enforcement, we have gone back and forth on this. We used to have some linter that would look at these things, but we removed it because it was too noisy with unhelpful suggestions. @sayboras Do you know if our GH Actions Go linter can pick this up?